code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -54,6 +54,9 @@
 	to_chat(user, "<span class='notice'>\The [src] is [anchored ? "":"not "]secured to the floor.</span>")
 
 /obj/structure/cult/functional/attackby(obj/I, mob/user, params)
+	if(HULK in user.mutations)
+		to_chat(user, "<span class='danger'>\"You cannot seem to manipulate this structure with your bulky hands!\"</span>")
```

Comment:
the `"` makes it seem as though the God is saying it? or whats this for? Probably shouldn't be there?